client/a8c-for-agencies/sections/reports/hooks/use-handle-report-delete.ts

```diff
@@ -13,35 +13,11 @@ export default function useHandleReportDelete() {
 	const dispatch = useDispatch();
 	const queryClient = useQueryClient();
 
-	// Reports query key so we optimistically update the reports list
+	// Reports query key so we can invalidate the reports list
 	const agencyId = useSelector( getActiveAgencyId );
 	const reportsQueryKey = getReportsQueryKey( agencyId );
 
 	const { mutate: deleteReport, isPending } = useDeleteReportMutation( {
-		// Optimistically update the reports list
-		onMutate: async ( { id }: { id: number } ) => {
```

Comment:
We handle this by refetching the reports. We end up in a weird state when it is the last report that is getting deleted. So, we will remove this logic to fix that.